Fixed #35072 -- Corrected Field.choices description in models topic. (#17663)
django/django: docs/topics/db/models.txt

    <Field.blank>`, the field will be required.

:attr:`~Field.choices`
    A :term:`sequence` of 2-value tuples, a :term:`mapping`, an :ref:`enumeration type <field-choices-enum-types>`,

>     or a parameter-less :term:`callable` which returns one of the previous formats

Reviewer: I'd leave only `a callable` without no further details:

```a parameter-less :term:`callable` which returns one of the previous formats``` -> ```a :term:`callable` ```

>     may be passed as choices for this field. If this is given, the default form widget

Reviewer: I'd keep the current wording
```suggestion
    to use as choices for this field. If this is given, the default form widget
```

    will be a select box instead of the standard text field,

>     and the widget will limit its choices to those that are given.

Reviewer: Wrap at 79 chars and add a missing `versionchanged` annotation:
```
.. versionchanged:: 5.0

    Support for mappings and callables was added.

```

        >>> p.get_shirt_size_display()
        'Large'

>     Here's how you can use enumeration classes to define ``choices`` in a concise

Reviewer: This change is not required, IMO. Please revert.
